### PR TITLE
[Stats Refresh] Detail list view: update view

### LINKED ISSUE #11165: [Stats Refresh] Detail list view: update view
From https://github.com/wordpress-mobile/WordPress-iOS/pull/11159#issuecomment-467779643:

1) For the tab state with no data present we can use our empty state.  Design:

detail 1 empty

2) include a grey background on the detail views. Design:

detail 1

### PR DESCRIPTION
**Pull request by ScoutHarris**
Fixes #11165 

To test:

---
- Go to Insights Comments or Followers.
- Select `View more`.
- Verify the table background is grey.

details_table

---
- On a site that has data for _one_ of the tabs for Insights Comments or Followers, select `View more`
  - Hogwarts > Followers is a good example.
- On the tab with no data, verify the view is:

no_data

### CODE REVIEW

**Comment by ScoutHarris** [comment]
@SylvesterWilmott -

---
with_data_portrait

---
no_data_portrait

---
with_data_landscape

---
no_data_landscape

File: WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift [hunk 1]
```diff
@@ -103,11 +113,35 @@ private extension TabbedTotalsCell {
...
+
+        guard showNoResults,
+            let superview = superview else {
+                return
+        }
+
+        noResultsViewController.view.frame = noResultsView.frame
+        noResultsViewController.view.frame.origin.y = 0
+        noResultsViewController.view.frame.size.height = superview.frame.height - filterTabBar.frame.height
+        noResultsView.addSubview(noResultsViewController.view)
```
Comment (stevebaranski) [inline WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift]: I see where we do the view controller containment dance for removal on line 117. Do we need to do the opposite on or about line 130?

File: WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift [hunk 2]
```diff
@@ -103,11 +113,35 @@ private extension TabbedTotalsCell {
...
+        let showNoResults = tabsData[filterTabBar.selectedIndex].dataRows.isEmpty && !limitRowsDisplayed
+        noResultsView.isHidden = !showNoResults
+
+        guard showNoResults,
+            let superview = superview else {
+                return
+        }
+
+        noResultsViewController.view.frame = noResultsView.frame
+        noResultsViewController.view.frame.origin.y = 0
```
Comment (stevebaranski) [inline WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift]: If we initialize line 127 with `bounds`, that should obviate the need for line 128.

File: WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift (hunk 1 above)
Comment (stevebaranski) [reply WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift]: I've struck through my comment above, as it became clear that we this class (`TabbedTotalsCell`) is not a view controller.

Comment (stevebaranski) [reply WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift]: I misunderstood that `TabbedTotalsCell` is sort-of overloaded. 

I see now that it's used to help render a card on the parent view (`SiteStatsInsightsTableViewController`), but on the child view (`SiteStatsDetailTableViewController`), it's _almost_ like a root view.

Carry on. 😄

File: WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift (hunk 2 above)
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift]: Nice one! Tis been changed! Thanks!

**Comment by SylvesterWilmott** [comment]
Looks good @ScoutHarris !

**Pull request merged**